src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -891,69 +858,63 @@ class BasicLayout extends React.PureComponent{
 
                 </div>
                 <div className="modal-backdrop fade show"> </div>
-            </React.Fragment>
-        }
-        else{
-            return
+            </React.Fragment>;
         }
     }
     getFormButtons(){
         //need to check if form is ready, if not just add the cancel button, or disable the create?
         return <React.Fragment>
             <button type="button" className="btn btn-secondary" data-dismiss="modal" onClick={this.onCancelNewWidget}>Cancel</button>
-            <button type="button" className="btn btn-primary" onClick={this.onCreateNewWidget} disabled={!this.state.FormReady}>Create</button>
-        </React.Fragment>
+            <button type="button" className="btn btn-primary" onClick={this.onCreateNewWidget} disabled={!this.state.formReady}>Create</button>
+        </React.Fragment>;
 
     }
     getBaseMapSelector(){
-        if(this.state.selectedWidgetType == "ImageCollection" || this.state.selectedWidgetType == "DualImageCollection" || this.state.selectedWidgetType == "imageAsset" || this.state.selectedWidgetType == "imageCollectionAsset") {
+        if(this.state.selectedWidgetType === "ImageCollection" || this.state.selectedWidgetType === "DualImageCollection" || this.state.selectedWidgetType === "imageAsset" || this.state.selectedWidgetType === "imageCollectionAsset" || this.state.selectedWidgetType === "ImageElevation") {
             return <React.Fragment>
                 <label htmlFor="widgetIndicesSelect">Basemap</label>
                 <select name="widgetIndicesSelect" value={this.state.WidgetBaseMap} className="form-control"
                         id="widgetIndicesSelect" onChange={this.onDataBaseMapSelectChanged}>
                     {this.baseMapOptions()}
                 </select>
-            </React.Fragment>
+            </React.Fragment>;
         }
-        else{return;}
     }
     baseMapOptions(){
-
-        let options = _.map(this.state.imagery, function (imagery) {
-            return <option key={imagery.id} value={imagery.id}> {imagery.title} </option>
+        return _.map(this.state.imagery, function (imagery) {
+            return <option key={imagery.id} value={imagery.id}> {imagery.title} </option>;
         });
-        return options;
     }
-    getDataType()
+    getDataTypeSelectionControl()
     {
-        if(this.state.selectedWidgetType == "-1")
+        if(this.state.selectedWidgetType === "-1")
         {
-            return
+            return <br/>;
         }
-        else if(this.state.selectedWidgetType == "statistics")
+        else if(this.state.selectedWidgetType === "statistics")
         {
-            if(this.state.FormReady != true){
+            if(this.state.formReady !== true){
```

Comment:
If you always want the state to end up true, you can just set it to true instead of checking it.